client/src/Components/Alerts/AlertList.js

```diff
@@ -1,17 +1,38 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import webHost from '../../config/web';
+import { Redirect, useHistory} from 'react-router-dom';
 
-import Card from '../../UIElements/Card'
+import webHost from '../../config/web';
+import api from '../../config/web';
+import Card from '../../UIElements/Card';
+import Modal from '../../UIElements/Modal';
+import CreateAlert from './CreateAlert';
 
 const AlertList = () => {
   const [list, setList] = useState([]);
   const [loading, setLoading] = useState(true);
+  const history = useHistory();
 
+  const deleteHandler = (id) => {
+    //delete alert using the id
+    axios.delete(`${api.URI}/${id}`)
+      .then((response) => {
+        console.log(response.data);
+        //reload page after deleting alert
+        window.location.reload();
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+
+  const updateHandler = (id) => {
+    history.push(`/alerts/update/${id}`);
```

Comment:
I love the attention to detail by adding this to the history

Reply:
Great